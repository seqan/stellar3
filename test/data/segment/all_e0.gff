1	Stellar	eps-matches	761	812	100	+	.	1;seq2Range=498,549;eValue=2.23482e-21;cigar=52M;mutations=
1	Stellar	eps-matches	266	316	100	+	.	1;seq2Range=251,301;eValue=7.34602e-21;cigar=51M;mutations=
1	Stellar	eps-matches	184	233	100	+	.	1;seq2Range=346,395;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	728	777	100	+	.	1;seq2Range=438,487;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	1247	1296	100	+	.	1;seq2Range=189,238;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	929	979	100	-	.	1;seq2Range=1101,1151;eValue=7.34602e-21;cigar=51M;mutations=
1	Stellar	eps-matches	1825	1874	100	-	.	1;seq2Range=1997,2046;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	1076	1125	100	-	.	1;seq2Range=1896,1945;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	708	757	100	-	.	1;seq2Range=1193,1242;eValue=2.4147e-20;cigar=50M;mutations=
1	Stellar	eps-matches	106	155	100	-	.	1;seq2Range=1411,1460;eValue=2.4147e-20;cigar=50M;mutations=
