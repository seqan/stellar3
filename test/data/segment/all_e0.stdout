I/O options:
  database file   : ref.fasta
  query file      : query_e0.fasta
  alphabet        : dna5
  output file     : segment/all_e0.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0 = (0/1)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500

Calculated parameters:
  k-mer length : 32
  s^min        : 50
  threshold    : 19
  distance cut : 50
  delta        : 16
  overlap      : 0

Loaded 1 query sequence.
Loaded 1 database sequence.

All matches resulting from your search have an E-value of: 
        2.4147e-20 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
    # SWIFT hits      : 5
    Longest hit       : 52
    Avg hit length    : 50
  1, complement
    # SWIFT hits      : 5
    Longest hit       : 51
    Avg hit length    : 50

# Eps-matches     : 10
Longest eps-match : 52
Avg match length  : 50
